Nvidia NIM Integration (#18964)
DataDog/integrations-core: nvidia_nim/README.md

# Agent Check: nvidia_nim

## Overview

This check monitors [NVIDIA NIM][1] through the Datadog Agent. 

## Setup

Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][3] for guidance on applying these instructions.

> **Note**: This check requires Agent v7.61.0+.

Reviewer: ```suggestion
**Requirements**:
- This check requires Agent v7.61.0+
- This check uses [OpenMetrics][10] for metric collection, which requires Python 3.
```
Putting all the requirements at the top is good practice so the user doesn't discover them one at a time. I'm not using a heading here cos I don't know if it would render in the tile.

### Installation

The NVIDIA NIM check is included in the [Datadog Agent][2] package. No additional installation is needed on your server.

### Configuration

> NVIDIA NIM provides Prometheus metrics indicating request statistics. By default, these metrics are available at http://localhost:8000/metrics. The Datadog Agent can collect the exposed metrics using this integration. Follow the instructions below to configure data collection from any or all of the components.

Reviewer 2: nit. Add the link to the nim docs if possible documenting this.

> **Note**: This check uses [OpenMetrics][10] for metric collection, which requires Python 3.

Reviewer: ```suggestion
```

> 1. Edit the `nvidia_nim.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your NVIDIA NIM performance data. See the [sample nvidia_nim.d/conf.yaml][4] for all available configuration options.

Reviewer: ```suggestion
To start collecting your NVIDIA NIM performance data:
1. Edit the `nvidia_nim.d/conf.yaml` file in the `conf.d/` folder at the root of your Agent's configuration directory. See the [sample nvidia_nim.d/conf.yaml][4] for all available configuration options.
```
